e2e/__tests__/__snapshots__/console.test.ts.snap

```diff
@@ -56,17 +98,22 @@ exports[`does not error out when using winston 2`] = `
 PASS __tests__/console.test.js
   ● Console
 
-    console.log node_modules/winston/lib/winston/transports/console.js:79
+    console.log
       {"level":"info","message":"Log message from winston"}
       
-    console.log node_modules/winston/lib/winston/transports/console.js:79
+      at Console.log (node_modules/winston/lib/winston/transports/console.js:79:23)
+    console.log
       {"message":"Info message from winston","level":"info"}
       
-    console.log node_modules/winston/lib/winston/transports/console.js:79
+      at Console.log (node_modules/winston/lib/winston/transports/console.js:79:23)
+    console.log
```

Comment:
Missing newline above this?

Reply:
Are these test cases similar to those non-verbose ones? Tailing new lines for stack frames seem to be removed in those cases, and winston's `transport.console` appends a newline after the message itself. 

One interesting thing I noticed is in non-verbose mode, new lines after the last stack frame are automatically truncated 🤔

Reply:
I think I've fixed this now